CONTRIBUTING.md

```diff
@@ -12,15 +12,15 @@ Contributions are very welcome on Karapace. When contributing please keep this i
 
 There is very little you need to get started coding for Karapace:
 
-- Use [one of the supported python versions](https://github.com/aiven/karapace/blob/master/setup.py)
-  documented in the `setup.py` classifiers.
+- Use [one of the supported python versions](https://github.com/Aiven-Open/karapace/blob/master/pyproject.toml)
+  documented in the `project:requires-python`.
 - Create [a virtual environment](https://docs.python.org/3/tutorial/venv.html) and install the dev dependencies in it:
 
 ```python
 python -m venv <path_to_venv>
 source <path_to_venv>/bin/activate
-pip install -r ./requirements/requirements-dev.txt
-pip install -e .
+pip install .
+pip install .[dev,typing]
```

Comment:
```suggestion
pip install -e .[dev,typing]
```

issue: Why is `-e` removed here? If it's removed, you would have to reinstall for any local changes to take effect for testing etc. I see there's no `src/` directory, which might mean that tests can see the non-installed local package instead, but that should really also be remediated to make sure tests run on the actually installed package.

Reply:
The move to `/src´ is good, I'll find some time to do it subsequent PR.